### ISSUE TITLE
`enter` gives output of `git status`

### ISSUE THREAD

**Issue by lode** [rtomayko/git-sh#38]
From thoughtbot/gitsh I loved the feature of using `enter` to get the current `git status`. It feels very natural and a handy shortcut.

(I do like how most commands, i.e. also arrow keys, are given to the normal systems prompt, that should not change in its base.)

**Comment by nikreiman** [comment]
I don't think that this is technically possible. It works in gitsh because that is a separate executable program, and can thus can handle `enter` separately. However, git-sh is just a set of shell aliases and extensions build on top of bash, and afaik bash does not offer a way to do this other than `PROMT_COMMAND`, which as the name suggests actually executes on every command (whether it was empty or not).
